### PR TITLE
Implemented a new feature of generating charts

### PR DESCRIPTION
**Pull request by OmalyaV**
A new feature is being created.
Graph generating according to user preference.

### CODE REVIEW

**Comment by OmalyaV** [comment]
@samithkavishke 
I have added a new feature to generate graphs with user preference.
A SQLite database is integrated to store the generated images temporarily.
However, when the extensio is deactivated, the images should be deleted too. That functionality is not working properly.

File: extension.js [hunk 1]
```diff
@@ -96,171 +97,42 @@ function activate(context) {
...
-                }
-            });
-        </script>
-    </body>
-    </html>
-    `
-}
-function runPythonScript(scriptName, filePath, callback) {
+function runPythonScript(scriptName, filePath, callback, options = []){
   const { spawn } = require("child_process")
```
Comment (samithkavishke) [inline extension.js:127]: Better to put all the imports at the top

File: media/script.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+//import { acquireVsCodeApi } from 'vscode';
+console.log("Script loaded");
```
Comment (samithkavishke) [inline media/script.js:2]: Remove redundant logging for debugs.

File: media/script.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                            columns.push(key);
+                            summaryHtml += '<tr>';
+                            summaryHtml += '<td>' + key + '</td>';
+                            summaryHtml += '<td>' + summaryData[key].count + '</td>';
+                            summaryHtml += '<td>' + summaryData[key].mean + '</td>';
+                            summaryHtml += '<td>' + summaryData[key].std + '</td>';
+                            summaryHtml += '<td>' + summaryData[key].min + '</td>';
+                            summaryHtml += '<td>' + summaryData[key]['25%'] + '</td>';
+                            summaryHtml += '<td>' + summaryData[key]['50%'] + '</td>';
+                            summaryHtml += '<td>' + summaryData[key]['75%'] + '</td>';
```
Comment (samithkavishke) [inline media/script.js:66]: better to put in an extracted helper function

**Review by samithkavishke — CHANGES_REQUESTED** [review]
Change those, I will look into the database problem

**Pull request merged**